src/quacc/atoms/skzcam.py

```diff
@@ -19,6 +19,38 @@
 has_chemshell = find_spec("chemsh") is not None
 
 
+def generate_pointcharge_file(
+        embedded_cluster: Atoms,
+        quantum_cluster_idx: list[int],
+        ecp_region_idx: list[int],
```

Comment:
I fixed this elsewhere but can you please use `indices` and not `idx` when it is a list? It is hard to understand that it is a list when the variable name is a singular index.